kernel/arch/dreamcast/include/dc/asic.h

```diff
@@ -24,12 +24,12 @@
 #include <sys/cdefs.h>
 __BEGIN_DECLS
 
-#include <arch/types.h>
+#include <stdint.h>
 
 /* All event codes are two 8-bit integers; the top integer is the event code
-   register to look in to check the event (and to acknolwedge it). The
-   register to check is 0xa05f6900+4*regnum. The bottom integer is the
-   bit index within that register. */
+    register to look in to check the event (and to acknolwedge it). The
+    register to check is 0xa05f6900+4*regnum. The bottom integer is the
+    bit index within that register. */
```

Comment:
These should not be spaced in an extra space like this... They should line up with the first character of the word `All` in the first line.
```suggestion
   register to look in to check the event (and to acknolwedge it). The
   register to check is 0xa05f6900+4*regnum. The bottom integer is the
   bit index within that register. */
```

Reply:
Done.